alchemiscale/interface/client.py

```diff
@@ -88,7 +88,7 @@ def get_scoped_key(self, obj: GufeTokenizable, scope: Scope) -> ScopedKey:
                 "Scope for a ScopedKey must be specific; it cannot contain wildcards."
             )
 
-    def check_exists(self, scoped_key: ScopedKey) -> bool:
+    def check_exists(self, scoped_key: Union[ScopedKey, str]) -> bool:
```

Comment:
Now that we're on Python 3.10+, we can switch these ugly `Union`s to `|`. I know there are more in the code base and this might not be the best place to do this, but figured I'd bring it up.

```suggestion
    def check_exists(self, scoped_key: ScopedKey | str) -> bool:
```